consensus/src/sync/live/queue.rs

```diff
@@ -82,32 +88,41 @@ impl<N: Network> ChunkAndSource<N> {
     }
 }
 
+/// Trait defining the behavior of a live sync queue; manages incoming blocks, tracks peers,
```

Comment:
TODO: I think this is not quite right, but I need to check this with someone else to come up with an adequate comment.